articles/role-based-access.md

```diff
@@ -48,7 +48,8 @@ GitOps is an API-only and write-only role that can be used on CI/CD pipelines.
 | Add user information from IdP to hosts\*                                                                                                   |          |            | ✅          | ✅    |        |
 | Create, edit, and delete labels                                                                                                            |          |            | ✅         | ✅    | ✅      |
 | View all software                                                                                                                          | ✅       | ✅         | ✅         | ✅    |         |
-| Add, edit, and delete software                                                                                                                    |          |           | ✅         | ✅    | ✅       |
+| Add, edit, and delete software                                                                                                             |          |           |            | ✅    | ✅       |
```

Comment:
This needs a description of some sort as someone looking at the table won't understand the difference between this and the below.

Also, now that this is team-specific I'm not sure why we need differing permissions here (particularly since team permissions are the same), so we likely don't need the extra row?